### ISSUE TITLE
Setup refreshVersions via a Plugin<Settings>

### ISSUE THREAD

**Issue by jmfayard** [jmfayard/refreshVersions#241]
NikkyAi asked on Slack whether instead of messing with the buildscript classpath, we could do something similar to the Gradle entreprise plugin

```kotlin
// settings.gradle.kts
plugins {
   id("de.fayard.refreshVersions.settings") version "0.9.XXXX"
}

refreshVersions {

}
```

- this would make the setup cleaner - no more messing with the classpath
- we could add configuration options via a Gradle extension instead of reinventing the wheel https://github.com/jmfayard/refreshVersions/issues/202

**Comment by jmfayard** [comment]
I've done an experiment in the `plugin-settings` branch and it seems to me that it would work!

https://github.com/jmfayard/refreshVersions/compare/develop...plugin-settings?expand=1

**Comment by LouisCAD** [comment]
We need to check if we can support the configuration that the bootstrap currently allows, and see if we can get it early enough for it to work for `buildscript`.

If yes, then we might be able to support both ways.

**Comment by LouisCAD** [comment]
Implemented in #280 and now merged into the develop branch.

**Comment by LouisCAD** [comment]
Finally released in 0.10.0
